test/validators.js

```diff
@@ -1279,8 +1279,8 @@ describe('Validators', () => {
       validator: 'isAlpha',
       args: ['el-GR'],
       valid: [
-        'αβγδεζηθικλμνξοπρςστυφχψω',
```

Comment:
For the tests, could you please add more tests other than overwriting the previous ones?

Reply:
Added tests instead of overwrite ✅